Penteract/Assets/Scripts/AIMeleeGrunt.cpp

```diff
@@ -87,7 +98,10 @@ void AIMeleeGrunt::Start() {
 	}
 
 	movementScript = GET_SCRIPT(&GetOwner(), AIMovement);
-
+	rightBladeCollider = movementScript->SearchRefenceInHierarchy(&GetOwner(),"RightBlade");
```

Comment:
By Mephistopheles, RefeREnce not Refence, pls.

Reply:
My fault :).